build-multiarch.sh

```diff
@@ -0,0 +1,63 @@
+TARGETIMAGE=${1:-target/image:ci}
+IS_RELEASE=${2:-no}
+WINBASE="mcr.microsoft.com/windows/nanoserver"
+OSVERSIONS=("1809" "1903" "1909" "ltsc2019" "2004" "20H2" "ltsc2022")
+MANIFESTLIST=""
+BUILDX_PUSH=""
+
+if [ "${IS_RELEASE}" = "yes" ]; then
+  export BUILDX_PUSH="--push";
+fi;
+
+echo "Building for Linux"
+docker buildx build \
+  --platform linux/amd64,linux/arm64,linux/ppc64le,linux/s390x,linux/386,linux/arm/v7,linux/arm/v6 \
+  ${BUILDX_PUSH} \
+  --pull \
+  --target linux \
+  -t ${TARGETIMAGE} \
+  .
+
+for VERSION in ${OSVERSIONS[*]}
+do
+  echo "Building Windows ${VERSION}"
+    docker buildx build \
```

Comment:
One thing that I noticed in my local workspace is that I needed to create a builder for windows/amd64, with
```
docker buildx create --platform windows/amd64 --name windows-builder --use
```
Otherwise, the windows platform was not available in my build context.

Do you know if the Docker installed in the ubuntu-latest comes with that support?

We could print `docker buildx ls` to verify it